rubin_sim/maf/stackers/generalStackers.py

```diff
@@ -67,24 +70,31 @@ def __init__(
         self.filterCol = filterCol
         self.airmassCol = airmassCol
         self.saturation_adu = saturation_e / gain
-        self.pixscale = 0.2
-        names = ["u", "g", "r", "i", "z", "y"]
-        types = [float] * 6
+        self.pixscale = pixscale
+        self.saturation_adu = saturation_e
```

Comment:
It's confusing having this line with line 72 above. 
Replace line 72 with line 74 and add a comment about why not using saturation_adu? 
Or alternatively, may make sense to replace saturation_adu entirely, as it seems like the rest of the calculations will only work with e- and not adu, given how the zero points are being calculated.